### ISSUE TITLE
Consider integrating into mainline org repo?

### ISSUE THREAD

**Issue by matthuszagh** [astahlman/ob-async#91]
I took a quick look at the source code for this and it seems that it's basically a copy/paste of `org-babel-execute-src-block`, with modifications for asynchronous execution. I think this would make it tough to stay in sync with org because any significant/relevant changes to `org-babel-execute-src-block` can break this package. I think it would make sense to merge this into the mainline so that any changes to `org-babel-execute-src-block` don't also need to be made here. Have you considered doing this?

**Comment by astahlman** [comment]
I agree that it would make sense. I don't have the free time to do it myself but I'd be delighted if someone else wants to take it up.

**Comment by matthuszagh** [comment]
I'm short on free time myself, but I'll see if I can find some time to do it. Thanks for being open to it!
